lib/settingPage.dart

```diff
@@ -0,0 +1,58 @@
+import 'package:flutter/material.dart';
+import 'package:flutter_settings_screens/flutter_settings_screens.dart';
+
+import 'themeUtil/change_theme_button_widget.dart';
+
+
+
+class SettingsPage extends StatelessWidget {
+    bool isSwitched = false;
+
+  SettingsPage({super.key});
+
+  @override
+
+  Widget build(BuildContext context) {
+
+    return  Scaffold(
+      appBar: AppBar(
+        title: const Text("Dark Mode"),
+       actions:[ ChangeThemeButtonWidget(),
+       ]
+      ),
+      body:SafeArea(
+        child: ListView(
+          padding: const EdgeInsets.all(24),
+          children: [
+            SettingsGroup(
+              title:'Account Settings',
+              children:
+              <Widget>[
+                buildLogout(),
+                buildDeleteAccount(),
+              ],
+           ),
+           
+            
+          ],
+        )
+      )
+
+        );
+  }
+}
+
+Widget buildLogout()=>SimpleSettingsTile(
+  title:'Logout',
+  subtitle: '',
+  leading:const Icon(Icons.logout)
+
+);
+
+Widget buildDeleteAccount()=>SimpleSettingsTile(
```

Comment:
Looks like we'll need to develop functionality if we decide to implement this

Reply:
You are correct, I only added these two widgets as placeholders until the time comes